ExtLibs/Controls/HUD.cs

```diff
@@ -31,9 +31,9 @@
 
 namespace MissionPlanner.Controls
 {
-    public class HUD2: HUD
+    public class graphicsObject: HUD

```

Comment:
can this be undone

Reply:
Good question. I did not explicitly do this. I wonder if MVS took over here.